src/components/layout/nav.tsx

```diff
@@ -66,41 +67,32 @@ const Nav: React.FC<NavProps> = (props: NavProps) => {
     // },
   ]
 
-  const [expanded, setExpanded] = useState<boolean>(false)
-
-  // const getInitialBannerState = () => {
-  //   if (typeof window === 'undefined') return true
-  //   return (
-  //     window.localStorage.getItem('bannerVisible') === null ||
-  //     window.localStorage.getItem('bannerVisible') !== 'hackathon'
-  //   )
-  // }
+  const bannerProps = {
+    title: 'hackathon',
```

Comment:
This shouldn't be hackathon anymore
```suggestion
    title: 'oct2020',
```